### PR TITLE
[16.0][ADD] edi_product_oca

### PR DESCRIPTION
**Pull request by OriolMForgeFlow**

### CODE REVIEW

**Review by JordiMForgeFlow — APPROVED** [review]
LGTM 👍🏼

File: edi_product_oca/models/product_product.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ProductProduct(models.Model):
+    _name = "product.product"
+    _inherit = ["product.product", "edi.exchange.consumer.mixin"]
+
+    edi_disable_auto = fields.Boolean(
```
Comment (simahawk) [inline edi_product_oca/models/product_product.py]: this field is already in the mixin

File: edi_product_oca/views/product_views.xml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        <field name="name">product.template.form.view - edi_product_oca</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_form_view" />
+        <field name="arch" type="xml">
+
+            <page name="inventory" position="after">
+                <page name="edi" string="EDI">
+                    <group
+                        name="edi_product_group"
+                        string="Electronic Data Interchange"
```
Comment (simahawk) [inline edi_product_oca/views/product_views.xml]:
```suggestion
```
this is a whole page for EDI, no need to be so verbose IMO

File: edi_product_oca/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2023 ForgeFlow S.L. (http://www.forgeflow.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Edi Product Oca",
```
Comment (simahawk) [inline edi_product_oca/__manifest__.py]:
```suggestion
    "name": "EDI Product"
```

File: edi_product_oca/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    "summary": """
+       Define EDI Configuration for Products""",
```
Comment (simahawk) [inline edi_product_oca/__manifest__.py]:
```suggestion
       EDI framework configuration and base logic for products""",
```

File: edi_product_oca/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    "version": "16.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ForgeFlow, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/edi-framework",
+    "depends": [
+        "product",
+        "edi_oca",
+    ],
+    "data": ["views/product_views.xml"],
+    "demo": [],
```
Comment (simahawk) [inline edi_product_oca/__manifest__.py]:
```suggestion
```

**Review by simahawk — CHANGES_REQUESTED** [review]
minor remarks

File: edi_product_oca/views/product_views.xml (hunk 2 above)
Comment (OriolMForgeFlow) [reply edi_product_oca/views/product_views.xml]: I personally preferred how it looked this way:

image

Better than how it looks for example in _edi_sale_oca_ without group name:

image

But nvm, already changed it! 👍🏼

**Review by simahawk — APPROVED** [review]

**Pull request merged**